backend/models/index.js

```diff
@@ -3,11 +3,20 @@ require('dotenv').config()
 const fs = require('fs')
 const path = require('path')
 const Sequelize = require('sequelize')
+
 const basename = path.basename(__filename)
 const db = {}
 
-const SqliteURI = `sqlite:${__dirname}/../db/dshop.db`
-const URI = process.env.DATABASE_URL || SqliteURI
+let URI
+if (process.env.NODE_ENV === 'test') {
+  // In test environment, always use SQLite with storage in a tmp directory.
+  URI = `sqlite:${process.env.TEST_TMP_DIR}/dshop.db`
```

Comment:
hm, I set up the tests to assign `DATABASE_URL`.  did that not work?

Reply:
It took me a while to figure what was going on. All my tests were failing... What happens with mocha is that setup.js gets evaluated AFTER the test files are loaded up. So the tests end up using whatever DATABASE_URL value is defined in your environment. And the overrides of DATABASE_URL in setup.js are not effective...